tests/records/test_jsonschema.py

```diff
@@ -162,11 +162,8 @@ def test_metadata(appctx):
 def test_resource_type(appctx):
```

Comment:
doubt: why appctx and not app? is there a diff? less heavy obj?

Reply:
Less heavy yes. `appctx` doesn't baggage with it the `database` and `es` fixtures. Because `app` does and they depend on `appctx`, `app` does push an application context as well, but is heavier. This is somewhat related to the above fixture discussion.